javascript/first-js-oop-project.md

```diff
@@ -56,6 +94,7 @@ description: "One day session on JavaSript"},
         // insert final html into #videos...
     $("#videos").html(html); });```
 
+- Create the html file name ***index.html***. 
```

Comment:
Change the sentence to `Create a HTML file named ***index.html***`

Reply:
@Kundan28 Please do the review. I have made the required changes.